transport/src/main/java/io/camunda/zeebe/transport/stream/impl/ClientStreamService.java

```diff
@@ -29,13 +32,30 @@ public ClientStreamService(final ClusterCommunicationService communicationServic
         new ClientStreamManager<>(
             new ClientStreamRegistry<>(),
             new ClientStreamRequestManager<>(communicationService, actor));
+    this.communicationService = communicationService;
+  }
+
+  @Override
+  protected void onActorStarted() {
+    // TODO: define a response type, so that broker can detect success or failure and take required
+    // action
+    communicationService.subscribe(
+        StreamTopics.PUSH.topic(),
+        MessageUtil::parsePushRequest,
+        clientStreamManager::onPayloadReceived,
+        actor::run);
```

Comment:
TODO: Here we may have to define a response type and send a response. Otherwise, it seems the handler is just a consumer and do not send exception back to the sender in case something fails.